lib/in/get-unused-packages.js

```diff
@@ -57,7 +72,14 @@ function checkUnused(currentState) {
             specials: getSpecialParsers(currentState)
         };
 
-        depcheck(currentState.get('cwd'), depCheckOptions, resolve);
+        let npmCheckRc = loadRcFile('npmcheck');
+
+        let depcheckOptions = {
```

Comment:
I don't think these are mutated?

```suggestion
        const npmCheckRc = loadRcFile('npmcheck');

        const depcheckOptions = {
```